docs/getting-started.md

```diff
@@ -1 +1,137 @@
-This is a placeholder for some docs
+## 1. Overview
+Kubeflow is a novel open source tool for Machine Learning workflow orchestration on Kubernetes.
+It has great powers, but deploying it may not be so easy, depending on how and where you deploy
+your Kubernetes. This tutorial will show you an easy way to deploy Kubeflow using MicroK8s, a
+lightweight version of Kubernetes, in a few simple steps.
+
+### What you'll learn
+
+* How to deploy MicroK8s on Ubuntu
+* How to deploy Kubeflow using MicroK8s
+* How to access your Kubeflow dashboard
+
+### What you'll need
+* Desktop or Virtual Machine with Ubuntu 16.04 or above
+* A minimum of 4 CPU, 14GB RAM, 50GB Disk (recommended 8 CPU, 16GB RAM, 60GB Disk)
+* Hyper V, if using Multipass on a Windows machine (not available on Windows 10 Home)
+* Some basic command-line knowledge
+
+
+## 2. Open Ubuntu terminal
+
+All we need to do here is access the terminal of the machine we want to deploy Kubeflow to. This
+is fairly straightforward.
+
+### Option 1: On Ubuntu machine
+
+1. Open terminal (CTRL + ALT + t)
+
+### Option 2: Accessing remote Ubuntu virtual machine
+
+1. Spin up a virtual machine with Ubuntu (on a server or cloud)
+
+1. Access the machine terminal using `ssh`:
+
+       $ ssh ubuntu@<machine-ip-address>
+
+### Option 3: On Windows or MacOS
+
+1. Download Multipass from [*multipass.run*][multipass]
+
+1. Launch Multipass instance with enough requirements:
+
+       $ multipass launch --name kubeflow --mem 16G --disk 50G --cpus 4
+
+1. Access Multipass instance:
+
+       $ multipass shell kubeflow
+
+[multipass]: http://multipass.run/
+
+
+## 3. Install Microk8s
+
+In this step you will install MicroK8s, a minimal, lightweight Kubernetes you can run and use on
+practically any machine. It can be installed with a snap:
+
+1. Install Microk8s:
+
+       $ sudo snap install microk8s --classic
+
+1. Verify installation:
+
+       $ sudo microk8s status --wait-ready
+
+
+## 4. Join admin group
+
+MicroK8s creates a group to enable seamless usage of commands which require admin privilege.
+
+1. To add your current user to the group and gain access to the `.kube` caching directory, run
+   the following two commands:
+
+       sudo usermod -a -G microk8s $USER
+       sudo chown -f -R $USER ~/.kube
+
+1. Exit and reënter the session for permissions to take effect.
+
+
+## 5. Deployment
+
+Finally we are able to deploy all the Kubernetes services behind Kubeflow. To do this we just need
+to:
+
+1. Deploy Kubernetes services:
+
+       $ microk8s enable dns dashboard storage
+
+1. Deploy Kubeflow using:
+
+       $ microk8s enable kubeflow
+
+1. Wait several minutes for all the services to be deployed, let's not interrupt the process. We
```

Comment:
Wait several minutes for all services to be deployed. Do not interrupt this process. We can follow the progress with the following command: